contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -1503,6 +1512,7 @@ export const Task = new Resource({
         }
       }
       deleteMoveTasks(change);
+      deleteDeleteNodeTasks(change);
```

Comment:
Should we also deleteCopyTasks?

Reply:
That's handled in the frontend code atm.